docs/src/how-to/install/team-feature-settings.md

```diff
@@ -10,6 +10,8 @@ By default Wire enforces a 2nd factor authentication for certain user operations
 
 If the `sndFactorPasswordChallenge` feature is enabled, a 6 digit verification code will be send per email to authenticate for additional user operations like e.g. for login, adding a new client, generating SCIM tokens, or deleting a team.
 
+Note the number of attempts to verify with a code is limited to 3 and requests for generating verification codes are rate limited. The default delay between consecutive requests is 5 minutes.
```

Comment:
```suggestion
After 3 attempts, the key is invalidated, and requests for generating new verification codes are rate limited. The default delay between two consecutive requests is 5 minutes.
```

Reply:
Also, I think 5 minutes is way too high as a default.  I would have picked 30 seconds, definitely not more than 60.  @comawill do you want to be the referee?

Reply:
Also, this paragraph is redundant, no?  You say everything in it again further down.